packages/db/src/meta/interface.ts

```diff
@@ -22,7 +24,7 @@ export interface Db<_C extends Collections> {
 }
 
 export interface ConnectOptions<_C extends Collections> {
-  workingDirectory: string;
+  directory?: string;
```

Comment:
Regarding my plan stated above, I'm thinking that we should try to remove this field entirely.